packages/local_auth/local_auth/android/src/test/java/io/flutter/plugins/localauth/LocalAuthTest.java

```diff
@@ -20,45 +24,95 @@
 import io.flutter.embedding.engine.plugins.lifecycle.HiddenLifecycleReference;
 import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
+
+import java.util.Collections;
+
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mock;
 
 public class LocalAuthTest {
-  @Test
-  public void isDeviceSupportedReturnsFalse() {
-    final LocalAuthPlugin plugin = new LocalAuthPlugin();
-    final MethodChannel.Result mockResult = mock(MethodChannel.Result.class);
-    plugin.onMethodCall(new MethodCall("isDeviceSupported", null), mockResult);
-    verify(mockResult).success(false);
-  }
 
-  @Test
-  public void onDetachedFromActivity_ShouldReleaseActivity() {
-    final Activity mockActivity = mock(Activity.class);
-    final ActivityPluginBinding mockActivityBinding = mock(ActivityPluginBinding.class);
-    when(mockActivityBinding.getActivity()).thenReturn(mockActivity);
+  private LocalAuthPlugin plugin;
+  @Mock private Context mockContext;
+  @Mock private Activity mockActivity;
+  @Mock private Lifecycle mockLifecycle;
+  @Mock private FlutterEngine mockFlutterEngine;
+  @Mock private DartExecutor mockDartExecutor;
+  @Mock private ActivityPluginBinding mockActivityBinding;
+  @Mock private MethodChannel.Result mockResult;
+  @Mock private FlutterPluginBinding mockPluginBinding;
 
-    Context mockContext = mock(Context.class);
-    when(mockActivity.getBaseContext()).thenReturn(mockContext);
+  @Before
+  public void setUp() {
+    plugin = new LocalAuthPlugin();
```

Comment:
Do not make the object under test part of the test fixture. It's an anti-pattern that makes understanding tests much more difficult.